_unittest/test_00_EDB.py

```diff
@@ -2754,3 +2756,9 @@ def test_143_add_layer_api_with_control_file(self):
             use_q3d=True,
         )
         assert setup.sweeps
+
+    def test_144_arc_data(self):
+        assert self.edbapp.nets["1.2V_DVDDL"].primitives[0].arcs
```

Comment:
Can we check the length of the list here, to make sure the arcs count is correct?
Just in case for instance if some arcs are duplicated or forgotten for any reason.